pytask_io/pytask_io.py

```diff
@@ -105,6 +75,81 @@ def __init__(self, *args, **kwargs):
         self.store_db = kwargs.get("store_db") or self.store_db
         self.workers = kwargs.get("workers") or self.workers
 
+        self._queue_client = None
```

Comment:
I dont understand why you need to replace this whole block of code which is containing all my past commits but now you are overriding it in your name...? why is this happening please?

Reply:
By `overriding` you mean creating a properties, which allow the validation that is being done? You know, if you really care about your name on the commit, you may as well change the commit author :) I do not..

Reply:
this it not how we work. We don't override other peoples code, thats just not polite....

Reply:
Ok, will stay with a single issue in the future. :)

Reply:
Thanks @MakuZo its not just my code, could be anyones, we have to respect the authors.. dont worry i made this mistake many times! thank you